src/app/shared/starts-with/date/starts-with-date.component.html

```diff
@@ -0,0 +1,25 @@
+<form class="w-100" [formGroup]="formData" (ngSubmit)="submitForm(formData.value)">
+  <div class="row">
+    <span class="col-12 font-weight-bold mb-1">
+      {{'browse.startsWith.jump' | translate}}
+    </span>
+    <div class="col-5 col-md-3">
+      <select class="form-control" (change)="setStartsWithEvent($event)">
+        <option [value]="-1" [selected]="!startsWith">
+          {{'browse.startsWith.choose_year' | translate}}
+        </option>
```

Comment:
Please also consider the support for the month as in JSPUI (http://demo.dspace.org/jspui/browse?type=dateissued&sort_by=2&order=ASC&rpp=20&etal=-1&year=2019&month=1&starts_with=).

Reply:
Added a dropdown to support adding a month to the startsWith parameter.